awscli/customizations/paginate.py

```diff
@@ -39,19 +39,37 @@
 STARTING_TOKEN_HELP = """
 <p>A token to specify where to start paginating.  This is the
 <code>NextToken</code> from a previously truncated response.</p>
+<p>For more information, see <a
```

Comment:
I'm not sure I want to link to this page as it is a bit out of date. If it were updated I may be inclined, but I would prefer that it read "for detailed examples".